extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -380,6 +382,9 @@ private static Table readTableInternal(@NotNull final File source,
         final BasicFileAttributes sourceAttr = readAttributes(sourcePath);
         if (sourceAttr.isRegularFile()) {
             if (sourceFileName.endsWith(PARQUET_FILE_EXTENSION)) {
+                if (instructions.isRefreshing()) {
+                    throw new IllegalArgumentException("Unable to have a refreshing single parquet file");
```

Comment:
Is this worth throwing an exception for? We can just ignore the setting.

Reply:
I would think that the user is expecting a ticking partition table whenever they set `instructions.isRefreshing()`. 

I'm not the biggest fan of how `readTable` branches to all these different paths depending on if it's a file, or a directory, or if there is a file in a directory, etc... but it _does_ make it a convenient entrypoint...

Reply:
This `readTable` is intended to be "easy to use". It has received a lot of positive feedback. The explicit, "tell us exactly what to do" option still exists in the public interface.

I suppose I can get behind your exception. I just think instructions tend to get copy-pasted a lot.